Kite SDK/PSPrintSDK/OLKiteABTesting.m

```diff
@@ -171,10 +172,25 @@ - (void)setupHidePriceTest{
                                 }];
 }
 
+- (void)setupOfferPayPalTest{
+    NSDictionary *experimentDict = [[NSUserDefaults standardUserDefaults] objectForKey:kOLKiteABTestOfferPayPal];
+    if (!experimentDict) {
+        experimentDict = @{@"Yes" : @0, @"No" : @1};
```

Comment:
I put that in deliberately because I thought that not having PayPal by default (even 0.5 of the time) carries some inherent risk that all our other AB tests don't. But I don't have arguments other than that so feel free to change it.


--
Konstadinos Karayannis

On 23 Jul 2015 5:26 PM +0300, Deon Botha<notifications@github.com>, wrote:
>  
> InKite SDK/PSPrintSDK/OLKiteABTesting.m(https://github.com/OceanLabs/iOS-Print-SDK/pull/202#discussion_r35326471):
>  
> >@@ -171,10 +172,25 @@ - (void)setupHidePriceTest{>}];>}>>+- (void)setupOfferPayPalTest{>+ NSDictionary *experimentDict = [[NSUserDefaults standardUserDefaults] objectForKey:kOLKiteABTestOfferPayPal];>+ if (!experimentDict) {>+ experimentDict = @{@"Yes" : @0, @"No" : @1};
>  
> I'm sure this should be 0.5 / 0.5 . Don't worry I'll change it when I merge but call me out if I'm wrong
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/OceanLabs/iOS-Print-SDK/pull/202/files#r35326471).
>  
>  
>